lib/worker-pipeline.js

```diff
@@ -105,9 +115,13 @@ class AssetComputeWorkerPipeline {
             input.type = SOURCE_TYPE_MAP[type] || (type && type.toLowerCase());
         }
 
-        if(!input.type || !output.type) {
-            // plan alg will throw error and send io events
-            console.log("Pipeline won't be able to find a plan and will throw error");
+        if (!input.type) {
+            //TODO should throw ClientError ?
```

Comment:
please keep comment:
```
// plan alg will throw error and send io events
```
this is the current behavior

Reply:
Note that you need to clarify the code if you need a comment to describe the behavior. The pipeline here should not worry about if/why the plan algorithm will do something.
I put back your comment, but please please clarify the behavior. Because if we rely on comments, that is a major readability issue for the code.